.github/PULL_REQUEST_TEMPLATE/new_loader.md

```diff
@@ -0,0 +1,32 @@
+### Title
+ 
+ Please use the following title: "Adding loader for MyDATASET". If your pull request is work in progress, change your title to "[WIP] Adding loader for MyDATASET" to avoid reviews while the loader is not ready.
+
+### Description
+
+Please include the following information in the description of the dataset:
```

Comment:
Maybe say where - like "include the following information at the top level docstring for the dataset's module `mydataset.py`